webapp/lib/charts.dart

```diff
@@ -180,10 +180,10 @@ class DailyTimeseriesLineChartView {
       chart.options.scales.xAxes[0].time = (new chartjs.TimeScale()
                                               ..displayFormats = new chartjs.TimeDisplayFormat(hour: 'D/MM hA'));
     }
-    if (upperLimit != null) {
+    if (yUpperLimit != null) {
       chart.options.scales.yAxes[0].ticks = (new chartjs.LinearTickOptions()
                                               ..beginAtZero = true
-                                              ..max = upperLimit);
+                                              ..max = yUpperLimit);
```

Comment:
I think it's worth implementing the functionality for `yLowerLimit` as well, even if we're not using it for now? otherwise it'll be a bit confusing next time we try to use the lower limit and it's not working.

I see that we already have the zero default we start the y axis at zero by using the `beginAtZero` option. So I suggest giving `yLowerLimit` the default value of 0 in the method header to replace `beginAtZero`, and then the code here would become something like: 
```
      chart.options.scales.yAxes[0].ticks = (new chartjs.LinearTickOptions()
                                              ..min = yLowerLimit
                                              ..max = yUpperLimit);
```
(not tested, so please make sure that this suggestion works!)

You should also check that `max = yUpperLimit` will work properly if `yUpperLimit` is `null` (ie. the default value), otherwise we need to maintain the if check.

Reply:
I've taken a similar approach I used in setting the min and max for the x-Axis. `yUpperLimit` cannot be null and so I have to use an `if` to check for null. It messes up the chart values. While I could the null safe operator as I did for the x-Axis it seems to not work here. Good suggestion on setting the `yLowerLimit` param to `0` so I don't have to null check it.